db/migrations/20201109152715_add_screenshots_table.js

```diff
@@ -0,0 +1,23 @@
+const { addUpdateTsTrigger } = require("../util")
+
+exports.up = async (knex) => {
+  await knex.schema.createTable("screenshots", (table) => {
+    table.increments()
+    table.string("name").notNullable()
```

Comment:
Should we include `file_extension`?

Reply:
At the moment we only save `png` files. I'm a fan of not adding unnecessary info to our databases. We can always use a migration script to add a `file_extension` column.